node/core/backing/src/tests/prospective_parachains.rs

```diff
@@ -1561,3 +1561,182 @@ fn occupied_core_assignment() {
 		virtual_overseer
 	});
 }
+
+#[test]
+fn seconding_check_request_notify() {
```

Comment:
re my comment on also re-running the `seconding` check on new active leaves, that should also be tested alongside this.